sql/mysqld.cc

```diff
@@ -5246,6 +5248,13 @@ static int init_server_components()
   error_handler_hook= my_message_sql;
   proc_info_hook= set_thd_stage_info;
 
+  /*
+   Print source revision hash, as one of the first lines, if not the 
+   first in error log, for troubleshooting and debugging purposes
+  */
+  sql_print_information("Starting MariaDB %s source revision %s", server_version, SOURCE_REVISION);
```

Comment:
Can you wrap this so the line length is under 80 characters and the wrapped arguments are indented to the '(' + 1 level.